core/src/test/java/org/apache/iceberg/rest/requests/TestCreateTableRequest.java

```diff
@@ -69,7 +69,7 @@ public void testRoundTripSerDe() throws JsonProcessingException {
         "\"fields\":[{\"name\":\"id_bucket\",\"transform\":\"bucket[16]\",\"source-id\":1,\"field-id\":1000}]}," +
         "\"order\":{\"order-id\":1,\"fields\":" +
         "[{\"transform\":\"identity\",\"source-id\":2,\"direction\":\"asc\",\"null-order\":\"nulls-last\"}]}," +
-        "\"properties\":{\"owner\":\"Hank\"},\"stageCreate\":false}";
+        "\"properties\":{\"owner\":\"Hank\"},\"stage-create\":false}";
```

Comment:
There's currently one failure `TestCreateTableRequest#testReoundTripSerDe`, that will be fixed once either https://github.com/apache/iceberg/pull/5135 or https://github.com/apache/iceberg/pull/5133 is merged.

```
TestCreateTableRequest > testRoundTripSerDe FAILED
    org.junit.ComparisonFailure: The serialized version of the Java record should match the original JSON expected:<...wner":"Hank"},"stage[-c]reate":false}> but was:<...wner":"Hank"},"stage[C]reate":false}>
        at org.junit.Assert.assertEquals(Assert.java:117)
        at org.apache.iceberg.rest.RequestResponseTestBase.assertRoundTripSerializesEquallyFrom(RequestResponseTestBase.java:90)
        at org.apache.iceberg.rest.requests.TestCreateTableRequest.testRoundTripSerDe(TestCreateTableRequest.java:83)
```

I've updated this for now so that this test failure won't cause issues and then will rebase when one of those PRs are merged.